framework/src/multiapps/MultiApp.C

```diff
@@ -516,22 +523,39 @@ MultiApp::fillPositions()
 void
 MultiApp::preTransfer(Real /*dt*/, Real target_time)
 {
-  // First, see if any Apps need to be Reset
-  if (!_reset_happened && target_time + 1e-14 >= _reset_time)
+  // Get a transient executioner to get a user-set tolerance
+  Real timestep_tol = 1e-13;
```

Comment:
Is there a reason why this was changed from `1e-14`?

Reply:
default s changed for transient multiapps too. It s meant to be overriden anyway by what is set for the executioner